charts/stateless-dns/values.yaml

```diff
@@ -116,8 +116,16 @@ externalDNS:
   interval: 30s  # The amount of time between external-dns runs.
   triggerLoopOnEvent: true  # Make external-dns to act by reacting to cluster events besides the polling with the interval above.
   policy: sync  # How DNS records are synchronized between sources and providers. sync will delete entries created by external-dns, it could also destroy entries that are not owner by external-dns if at some point it mark it as owned. upsert-only will create entries but will prevent the deletion of them.
-  txtOwnerId: ""  # Each of the entris added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. This is an ID that will be added to that TXT entry.
-  txtPrefix: ""  # Each of the entris added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. The TXT entry name will be the entry that is being created plus this prefix.
+  txtOwnerId: ""  # Each of the entries added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. This is an ID that will be added to that TXT entry.
+  # Each of the entries added by external-dns will create a TXT entry in the DNS too to mark external-dns as it owner. The TXT entry name will be the entry that is being created plus this prefix.
+  # external-dns needs to add to the TXT registry the record type (A, NS, etc.) of the original record in order to have
+  # separate TXTs for different record types with the same name. If the special token `%{record_type}` is not specified
+  # in this prefix, external-dns will add it automatically to the domain name, e.g. `test-ns.local` for `test.local NS`.
+  # If this happens, PowerDNS would reject this record as it most likely will not have a zone for `test-ns.local`.
+  # For stateless-dns to work, this prefix must:
+  # 1. End with a dot (`.`), so all TXT records belong to the top level zone.
+  # 2. Contain the `%{record_type}` token before that dot.
+  txtPrefix: "%{record_type}.stateless-dns."
```

Comment:
Just a NIT. The format that this `values.yaml` has is not the one that is used with [helm-docs](https://github.com/norwoodj/helm-docs) but the one from [frigate](https://github.com/rapidsai/frigate).

The comment of the variable should be in the same line as the variable. At least something that points to the `values.yaml` for more information.

It is a NIT because I was exploring it, I am not sure if I am going to use it or switch to helm-docs but I would love to keep the same structure in the whole file. If I implement Frigate I can fix in in the future.